core.json

```diff
@@ -0,0 +1,50 @@
+{
+    "$schema": "http://json-schema.org/draft-08/schema#",
+    "$id": "https://json-schema.org/draft-08/core",
+    "$recursiveAnchor": true,
+    "$vocabulary": {
+        "https://json-schema.org/draft-08/vocabularies/core": true
+    },
+
+    "title": "Core vocabulary meta-schema",
+    "type": ["object", "boolean"],
+    "properties": {
+        "$id": {
+            "type": "string",
+            "format": "uri-reference"
+        },
+        "$schema": {
+            "type": "string",
+            "format": "uri"
+        },
+        "$ref": {
+            "type": "string",
+            "format": "uri-reference"
+        },
+        "$recursiveRef": {
+            "type": "string",
+            "format": "uri-reference"
+        },
+        "$recursiveAnchor": {
+            "type": "boolean"
+        },
+        "$vocabulary": {
+            "type": "object",
+            "propertyNames": {
+                "type": "string",
+                "format": "uri"
+            },
+            "additionalProperties": {
+                "type": "boolean"
+            }
+        },
+        "$comment": {
+            "type": "string"
+        },
+        "$defs": {
```

Comment:
In the PR where we added this, it was determined that we want to keep `definitions` for backward compatibility.  Would that go here or in another meta-schema component?

Reply:
@gregsdennis no, `definitions` and also `dependencies` will only appear in the draft-08 versions of the  existing "schema" and "hyper-schema" documents.  Backwards compatibility here means "backwards compatibility for users of the meta-schemas that used to contain them".

Those keywords are not part of the proper vocabularies.  There were no formal vocabularies before draft-08, so there is nothing requiring compatibility.

Reply:
@gregsdennis we could also make a tiny compatibility vocabulary for the "schema" meta-schema that contained those two keywords, and then implementations that want to offer compatibility can check for that vocabulary.

I'm kind of waiting to see if this approach gets a green light at all before sorting out the details of how many vocabularies we have and what's in them.